docs/01_general/04_subscriptions.md

```diff
@@ -5,4 +5,140 @@ path: /docs/subscriptions
 
 # Subscriptions
 
-Documentation coming soon
+In GraphQL you can use subscriptions to stream data from a server. To enable
+this with Strawberry your server must support ASGI and websockets.
+
+This is how you define a subscription-capable resolver:
+
+```python
+import asyncio
+
+import strawberry
+
+@strawberry.type
+class Subscription:
+    @strawberry.subscription
+    async def count(self, target: int = 100) -> int:
+        for i in range(target):
+            yield i
+            await asyncio.sleep(0.5)
+
+schema = strawberry.Schema(subscription=Subscription)
```

Comment:
I think you'll need to pass a `query` otherwise it will raise an exception.

Reply:
It seems the [mutations docs](https://github.com/strawberry-graphql/strawberry/blob/master/docs/01_general/03_mutations.md) have the same issue.

Reply:
@jkimbo thoughts on: https://github.com/strawberry-graphql/strawberry/pull/613 ?